src/facade/dragonfly_connection.cc

```diff
@@ -192,6 +195,7 @@ thread_local TrafficLogger tl_traffic_logger{};
 thread_local base::Histogram* io_req_size_hist = nullptr;
 
 thread_local const size_t reply_size_limit = absl::GetFlag(FLAGS_squashed_reply_size_limit);
+thread_local uint32 pipeline_wait_batch_usec = absl::GetFlag(FLAGS_pipeline_wait_batch_usec);
```

Comment:
I've neved initialized global variables from absl flags and never though if it's safe. This initialization happens before main? But it somehow works...? Also, hey are initialized in the order in which they are _defined_. No tricks in absl? Seems fragile

Reply:
Also given how rarely this changes, it can be just a global atomic - there won't be any read overhead

Reply:
The whole absl flag caching scheme is still somewhat blurry. I agree we shouldn't use them directly in hot paths, but we currently do somethimes (see MGET 🤷🏻‍♂️)